framer/scripts/framerConfig.js

```diff
@@ -314,6 +314,7 @@ export const componentSettings = {
       'gutterBottom',
       'internalDeprecatedVariant',
       'paragraph',
+      'variant',
```

Comment:
It's definitely an important prop, maybe?

```suggestion
      // FIXME: `Union`
      'variant',
```

cc @mbrookes